src/scrape_up/Pixabay/pixabay.py

```diff
@@ -0,0 +1,142 @@
+import requests
+
+
+class Pixabay():
+    """
+    Class - `Pixabay`\n
+
+    | Methods                     | Details                                                                                              |
+    | --------------------------- | ---------------------------------------------------------------------------------------------------- |
+    | `.get_video()`              | Downloads the videos from pixaby to the local storage.                                               |
+    | `.get_photo()`              | Downloads the photos from pixaby to local storage.                                                   |
+
+
+    """
+    def __init__(self, verbose=True):
+        self.__api_key = '38504833-19606430bd8fde504120d1630'
```

Comment:
What change is this? This doesn't make the keys safe.

And most importantly, we are building an API alternative using web scraping and asking users to use API keys simultaneously. This doesn't match the motive.

Try building the scraper without using API keys, or else close the PR.

Reply:
@nikhil25803 actually the users would not have to use their api if they would use this scraper. This given api-key is self-sufficient to generate and store any number of images anytime from pixabay. 

Plus websites like amazon, Pixabay etc.. actually forbid scraping from their webpages. For same purpose, we gonna have to use an API in our web scraper if we are not using Selenium(which is also restricted by you to use). 

Nevertheless, I'd make it private for users to see, so that they would not be able to see/retrive it. @nikhil25803

Reply:
Still, it doesn't make any sense to use API keys in this package.

See, we have 1000+ lines of code to scrape data from GitHub, that call could have been done in some 100 lines of code using APIs. But due to problems like rate limit and all one can use this package. 

So where is the difference? Closing this PR, though you have done a great job. I'll recommend you add the code to your github as a public repository.